src/components/settings/SettingsLMSTab/ExistingLMSCardDeck.jsx

```diff
@@ -0,0 +1,153 @@
+import React, { useState } from 'react';
```

Comment:
good point, I'm not sure where the slow load times come from but also I'm very curious if this will run differently on stage/prod than it does on our local machines